### PR TITLE
Add incident report on DNS outage

### PR DESCRIPTION
**Pull request by evolutionise**
# About
This is an incident report for the recent DNS outage

# Release

N/A

# Blog image idea

I have reused the shield graphic from the last incident report.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://www.octopus.design/writing/writing-tldr/).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://www.octopus.design/writing/working-with-images/).
- [x] I've added alt text for all the images I've included (alt text describes image to people unable to see it, 125 characters max)
- [ ] One screenshot (if any are included) is suitable for Twitter. This screenshot should be 16:9 ratio and at least 1200 x 675 pixels. For screenshots from the Octopus Web Portal, please use Chrome and the browser’s zoom function to remove as much white space as possible. If screenshots are annotated (e.g., with boxes or arrows), please also provide a version that is unannotated for use on Twitter. Do not use dark mode for the Octopus Web Portal.

## How to review this PR
For review by @michaelnoonan @TeganAli @octoliamp 

## Publication date
For publication today, based on discussion with Tegan.

### CODE REVIEW

File: blog/2023-02/incident-dns-outage/index.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+_(All dates and times below are shown in UTC)_
+
+### Wednesday, January 25, 2023
+
+**_02:00_** An Octopus engineer deployed a change to the `octopus.com` and `octopusdeploy.com` domains, moving the associated routing between Azure Front Door profiles as part of a migration project.
+
+**_02:39_** The engineer switched the AFD WAF from prevention mode to detection mode due to it blocking valid traffic.
+
+**_07:05 _** A network outage in Azure obscured the issue - see our [Octopus Cloud connectivity disruption report](https://octopus.com/blog/cloud-connectivity-disruption-report-learnings).
```
Comment (evolutionise) [inline blog/2023-02/incident-dns-outage/index.md:54]: Minor but there's a space after the time that is breaking the rendering in github, don't know what the impact will be on the blog

File: blog/2023-02/incident-dns-outage/index.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+### What happened
+
+AFD does not update all of its nodes when a routing change happens, so some AFD nodes were still directing traffic to the incorrect web application. The incorrect application was what the affected endpoint was resolving to before the migration. Without control of a CNAME record, AFD was unable to authoritatively update the flattened A records.
+We set up a CNAME record on AFD in order to give them authoritative control of all requests on that subdomain. The incorrect IP address resolution behaviour on the affected endpoint continued.
+The incorrect routing inside AFD stopped after Azure advised us they had made internal changes.
+
+## Remediation
+
+We believe there were two red herrings which turned out not to be contributing factors:
```
Comment (TeganAli) [inline blog/2023-02/incident-dns-outage/index.md]:
```suggestion
We believe there were 2 red herrings which turned out not to be contributing factors:
```

Comment (TeganAli) [inline blog/2023-02/incident-dns-outage/index.md]:
```suggestion
We believe there were 2 red herrings which turned out not to be contributing factors:
```

File: blog/2023-02/incident-dns-outage/index.md [hunk 3]
```diff
@@ -120,102 +118,111 @@ Engineers focused on the failing route in the AFD profile.
...
 
-We believe there were two red herrings which turned out not to be contributing factors:
+There were 2 red herrings that weren’t contributing factors:
 
 - The 404 errors from the WAF
-- The CNAME flattening on the apex domain.
-  In practice, removing the proxy and switching to AFD’s WAF are irrelevant to this issue - they are included here because they created a red herring when attempting to determine the causes.
+- The CNAME flattening on the apex domain
+
+In practice, removing the proxy and switching to AFD’s WAF is irrelevant to this issue - we included them here because they created a red herring when attempting to determine the causes.
```
Comment (evolutionise) [inline blog/2023-02/incident-dns-outage/index.md:198]: Sorry I think I bungled this line - I was editing it and I think it should have been something more like
"In practice, removing the proxy and switching to AFD’s WAF were unrelated to this issue. However they created significant noise which obscured the real problem."

**Review by alexrolleyoctopus — APPROVED** [review]

**Review by michaelnoonan — APPROVED** [review]

**Pull request merged**